### PR TITLE
TASK#119 create add member button at the top of the screen

### LINKED ISSUE #119: CREAR BOTÓN AÑADIR SOCIO EN LA PARTE SUPERIOR DE LA PANTALLA/CREATE ADD MEMBER BUTTON AT THE TOP OF THE SCREEN
PROJECT TASK TEMPLATE

### EXPLANATION 

Después de haber seleccionado la opción Añadir, se desplegara una vista en la cual encontramos en la parte superior un botón con la opción "Añadir socio" que al darle clic nos lleva a la vista donde se piden los datos del nuevo socio.

After selecting the Add option, a view will be displayed in which we find at the top a button with the option "Add member" that when clicked takes us to the view where the new member's data is requested.

### SCREENSHOTS

Image

Image

### CHECKLIST

Al seleccionar la opción Añadir, se debe abrir una vista donde aparezca el botón  añadir socio en la parte superior de la pantalla, para agregar la información de un nuevo socio.

When selecting the Add option, a view should open where the add member button appears at the top of the screen to add a new member's information.

### USE CASES

En la opción añadir socio, al pasar el cursor debe tener una reacción ( subrayar).

In the member add option, when you mouse over, you should have a reaction (underline).

### PR DESCRIPTION
**Pull request by JuanParraIV**
### Description

Please include a summary of the change.
USE CASES
En la opción añadir socio, al pasar el cursor debe tener una reacción ( subrayar).

In the member add option, when you mouse over, you should have a reaction (underline).

fixes # (issue)
(119)
#### Screenshots
Screenshot from 2022-08-30 17-45-41
If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Review by angelik0828 — APPROVED** [review]

**Review by Eghinner — CHANGES_REQUESTED** [review]
You can make the effect lighter. Actually the border makes the letters change position.

I recommend implementing this reference for the effect: https://tobiasahlin.com/blog/css-trick-animating-link-underlines

And save this utility in stylesutilities.scss

**Review by Israel-Laguan — APPROVED** [review]

**Comment by angelik0828** [comment]
## With the change it is working

image
image

**Review by angelik0828 — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

**Review by Eghinner — CHANGES_REQUESTED** [review]
should not have underline in the icon. 
chrome_DEF3JPYVXm
Hover over the entire button and have only the words underline.

**Comment by angelik0828** [comment]
##  ✔️ Test in chrome and firefox in both browsers works fine for me, some other details of the bug to help identify it

image
image

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
@JuanParraIV please check why "npm run build" is not working

File: styles/utilities.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.hover {
+  span {
```
Comment (Israel-Laguan) [inline styles/utilities.scss:2]: Not sure if this name (`.hover`) is fine, please check alternatives

Comment (JuanParraIV) [reply styles/utilities.scss:2]: the hover effect is working from ListHeader.module.scss. i dont know how reutilize classes in utilities.scss cant be import in the component

Comment (JuanParraIV) [reply styles/utilities.scss:2]: Screenshot from 2022-09-06 09-21-47

Comment (JuanParraIV) [reply styles/utilities.scss:2]: Screenshot from 2022-09-06 09-22-22

**Review by Israel-Laguan — APPROVED** [review]

File: styles/utilities.scss (hunk 1 above)
Comment (Israel-Laguan) [reply styles/utilities.scss:2]: @Eghinner please help here

Comment (Eghinner) [reply styles/utilities.scss:2]: @import utilities in global.scss

**Comment by Eghinner** [comment]
looks good @JuanParraIV do you want to add anything else?

**Review by Eghinner — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Comment by Eghinner** [comment]
can you please resolve conflicts?

**Comment by JuanParraIV** [comment]
when everything is in order merge this before #126

**Pull request merged**